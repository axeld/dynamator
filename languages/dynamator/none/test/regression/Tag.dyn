<dynamator language="none">
  <tag tag="title">
    <content>Success!</content>
  </tag>

  <tag tag="meta" name="Description">
    <attr name="content">
      <content>Success!</content>
    </attr>
  </tag>
  
  <id name="h1">
    <content>Specified by id</content>
  </id>

  <tag tag="h1">
    <content>Success!</content>
  </tag>

  <tag tag="body">
    <attr name="bgcolor">
      <discard/>
    </attr>
  </tag>

  <tag tag="p" with-attr="align">
    <discard/>
  </tag>
  
</dynamator>
