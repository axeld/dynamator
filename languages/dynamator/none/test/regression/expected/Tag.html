<html>
  <head>
    <title>Success!</title>
    <meta name="Description" content="Success!">
    <meta name="Keywords" content="Should not change">
  </head>
  <body>
    <h1 id="h1">Specified by id</h1>
    <h1>Success!</h1>
    <p>OK</p>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
